doc/use_cli_effectively.md

```diff
@@ -1,292 +1,3 @@
-
-   For other details, check out [Stefan's blog](https://blog.jhnr.ch/2018/05/16/working-with-azure-cli-behind-ssl-intercepting-proxy-server/).
-
-2. Disable the certificate check across Azure CLI by setting environment variable `AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1`. This is not safe, but good for a short period like capturing a network trace for a specific command and promptly turning it off when finished. This may not work for some data-plane commands due to underlying SDK limitations.
-
-## Concurrent builds
-
-If you are using az on a build machine, and multiple jobs can be run in parallel, then there is a risk that the login tokens are shared between two build jobs is the jobs run as the same OS user.  To avoid mix ups like this, set AZURE_CONFIG_DIR to a directory where the login tokens should be stored.  It could be a randomly created folder, or just the name of the jenkins workspace, like this ```AZURE_CONFIG_DIR=.```
-
-## Appendix
-
-### Windows batch scripts for saving to variables and using it later
-
-```batch
-ECHO OFF
-SETLOCAL
-FOR /F "tokens=* USEBACKQ" %%F IN (`az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv`) DO (
-    SET "vm_ids=%%F %vm_ids%"  :: construct the id list
-)
-az vm stop --ids %vm_ids% :: CLI stops all VMs in parallel
-```
-
-### Windows PowerShell scripts for saving to variables and using it later
-
-```powershell
-$vm_ids=(az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv)
-az vm stop --ids $vm_ids # CLI stops all VMs in parallel
-```
-
-### Windows batch scripts to loop through a list
-```batch
-ECHO OFF
-SETLOCAL
-FOR /F "tokens=* USEBACKQ" %%F IN (`az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv`) DO (
-    ECHO Stopping %%F
-    az vm stop --ids %%F
-)
-```
-
-### Windows PowerShell scripts to loop through a list
-```powershell
-$vm_ids=(az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv)
-foreach ($vm_id in $vm_ids) {
-    Write-Output "Stopping $vm_id"
-    az vm stop --ids $vm_id
-}
-```
-
-### CLI Environment Variables
-
-|  Environment Variable          | Description            |
-|--------------------------------|------------------------|
-| **AZURE_CONFIG_DIR**           | Global config directory for config files, logs and telemetry. If unspecified, defaults to `~/.azure`. |
-| **AZURE_EXTENSION_DIR**        | Extensions' installation directory. If unspecified, defaults to `cliextensions` directory within the global config directory. |
+The content of this article has been moved to Microsoft Docs [How-to use Azure CLI effectively](/cli/azure/use-cli-effectively).
```

Comment:
@dbradish-microsoft could you fix the link and make it work before we merge this PR?

Reply:
@jiasli, That is the correct link, but the doc hasn't been published yet.  I was going to do a #hold-off until the article arrived in azure-docs-cli, but I couldn't get past "Azure.azure-cli (Check the Format of Pull Request Title)".  I'll leave as-is until the link will work.

Reply:
This is because we have a [CI check for the PR title](https://github.com/Azure/azure-cli/blob/806e4318d058c3037566fa5a7a07cfc99d1ff51d/doc/authoring_command_modules/README.md#format-pr-title). I have fixed the title to comply with the requirement and re-run the CI. Don't worry. 😉